### DISCUSSION TITLE
TOC not working in page but works fine in posts

### DISCUSSION THREAD

**Discussion by Edz0ro** [hugo-fixit/FixIt#302] [Q&A] [answered]
hugo v0.110.0-e32a493b7826d02763c3b79623952e625402b168+extended windows/amd64 BuildDate=2023-01-17T12:16:09Z VendorInfo=gohugoio

Hi ! I didn't find many informations in the docs about TOC. 

I already checked that toc is enable in my site config as well as in the front matters of my content but I can't find a way to make it work in my pages although it works fine in my posts. 

So please can someone explain to me how can I solve this ?

 I feel like I'm missing something ^^'

**Comment by Lruihao** [comment, marked as answer]
Just add a front matter：

```
type: posts
```

**Comment by Edz0ro** [comment]
Thanks. 

I thought I already tried that but it was not the case.
